src/main/java/org/jitsi/videobridge/cc/RtpState.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright @ 2019 8x8, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.videobridge.cc;
+
+/**
+ * This class represents the RTP state and is used in adaptive track context
+ * switches (see {@link AdaptiveTrackProjection}).
+ *
+ * @author George Politis
+ */
+public class RtpState
+{
+    /**
+     * The SSRC of the RTP stream that this information pertains to.
+     */
+    public final long ssrc;
+
+    /**
+     * The highest sent RTP timestamp.
+     */
+    public final long maxTimestamp;
+
+    /**
+     * The highest sent sequence number.
+     */
+    public final int maxSequenceNumber;
+
+    /**
+     * The number of transmitted bytes so far.
+     */
+    public final long transmittedBytes;
```

Comment:
what do we use this and transmittedPackets for?

Reply:
where?

Reply:
It's the responsibility of the adaptive track context to update these values in the SRs, so you will find these updates in GenericAdaptiveTrackProjectionContext and VP8AdaptiveTrackProjectionContext, along with timestamp updates in order for the receiving endpoint to be able to correctly map RTP timestamps to NTP timestamps.